tests/claimWin.test.ts

```diff
@@ -38,12 +44,10 @@ describe("Wins claiming and WinClaimed Event", () => {
 
     await claimWin(poolAccountKey, optionAccountKey, entryAccountKey, user);
 
-    try {
+    // account deleted after a successful win claim
+    await expect(async () => {
```

Comment:
I think you don't need the async `() => {` await wrapper, you can simply do `await expect(claimWin(poolAccountKey, optionAccountKey, entryAccountKey, user)).rejects`

Reply:
Ohh yeah you're right. Let me modify this throughout